llvm/lib/Transforms/Instrumentation/InstrProfiling.cpp

```diff
@@ -170,6 +170,29 @@ cl::opt<bool> SkipRetExitBlock(
     "skip-ret-exit-block", cl::init(true),
     cl::desc("Suppress counter promotion if exit blocks contain ret."));
 
+static cl::opt<bool> SampledInstr("sampled-instr", cl::ZeroOrMore,
```

Comment:
'instr' can be confused with 'instruction'. We should just spell it out as 'sampled-instrumentation'.

Reply:
ack.